src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/SecurityController.php

```diff
@@ -0,0 +1,337 @@
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
+        return $this->redirectToRoute('admin_security_sessions_customer_list');
+    }
+
+    /**
+     * Bulk delete customer session.
+     *
+     * @AdminSecurity(
+     *     "is_granted('delete', request.get('_legacy_controller')~'_')",
+     *     message="You do not have permission to edit this."
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkDeleteCustomerSessionAction(Request $request): RedirectResponse
+    {
+        $sessionIds = $request->request->get('security_session_customer_bulk');
+
+        try {
+            $deleteSessionsCommand = new BulkDeleteCustomerSessionsCommand($sessionIds);
+
+            $this->getCommandBus()->handle($deleteSessionsCommand);
+
+            $this->addFlash('success', $this->trans('Successful deletion', 'Admin.Notifications.Success'));
+        } catch (CoreException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
+        return $this->redirectToRoute('admin_security_sessions_customer_list');
+    }
+
+    /**
+     * Bulk delete employee session.
+     *
+     * @AdminSecurity(
+     *     "is_granted('delete', request.get('_legacy_controller')~'_')",
+     *     message="You do not have permission to edit this."
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkDeleteEmployeeSessionAction(Request $request): RedirectResponse
+    {
+        $sessionIds = $request->request->get('security_session_employee_bulk');
+
+        try {
+            $deleteSessionsCommand = new BulkDeleteEmployeeSessionsCommand($sessionIds);
+
+            $this->getCommandBus()->handle($deleteSessionsCommand);
+
+            $this->addFlash('success', $this->trans('Successful deletion', 'Admin.Notifications.Success'));
+        } catch (CoreException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
+        return $this->redirectToRoute('admin_security_sessions_employee_list');
+    }
+
+    /**
+     * Get human readable error for exception.
+     *
+     * @param Exception $e
+     *
+     * @return array
+     */
+    protected function getErrorMessages(Exception $e): array
+    {
+        return [
+            SessionNotFoundException::class => $this->trans(
+                'The object cannot be loaded (or found).',
+                'Admin.Notifications.Error'
+            ),
+            CannotDeleteCustomerSessionException::class => $this->trans(
+                'An error occurred while deleting the object.',
+                'Admin.Notifications.Error'
+            ),
+            CannotBulkDeleteCustomerSessionException::class => $this->trans(
+                '%s: %s',
```

Comment:
Hi, I've just noticed this string doesn't have a domain and falls in the 'messages'. I'm not sure to understand what it stands for and where it will appear though... Could you tell me more so I can think of an appropriate domain, please? 
Thank you in advance

Reply:
it's about the usage of `:`:
In french `Il dit : ça va ?`
In English `He says: Hey bro!`

Reply:
It sounds pretty general, I'd go with the domain Admin.Global

Reply:
```suggestion
                '%s: %s','Admin.Global'
```